.github/workflows/xpu-max1100.yml

```diff
@@ -0,0 +1,59 @@
+name: xpu-max1100
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+  pull_request:
```

Comment:
@Liangliang-Ma - I'm not sure we would want to run this on every PR at the start?  Perhaps if we can add it to just run on PRs when certain files are modified?

Also it appears the runner isn't created yet.

Reply:
Thanks for review. I have added a initial scope for the CI.